teams/teams-ps/teams/Set-CsPhoneNumberAssignment.md

```diff
@@ -38,6 +38,10 @@ To remove a phone number from a user or resource account, use the [Remove-CsPhon
 If the cmdlet executes successfully, no result object will be returned. If the cmdlet fails for any reason, a result object will be returned that contains a Code string parameter
 and a Message string parameter with additional details of the failure.
 
+**Note**: In Teams PowerShell Module 4.2.1-preview and later we are changing how the cmdlet reports error. Instead of using a result object, we will be generating an

```

Comment:
```suggestion
**Note**: In Teams PowerShell Module 4.2.1-preview and later we are changing how the cmdlet reports errors. Instead of using a result object, we will be generating an

```